source/extensions/filters/http/buffer/buffer_filter.cc

```diff
@@ -113,8 +113,7 @@ void BufferFilter::onDestroy() {
 }
 
 void BufferFilter::onRequestTimeout() {
-  Http::Utility::sendLocalReply(*callbacks_, stream_destroyed_, Http::Code::RequestTimeout,
```

Comment:
I think the only reason we are tracking stream_destroyed_ is because of this call. Can you remove that member variable? this comment likely applies to other filters also.

Reply:
OK!